iothub/device/src/Transport/Amqp/AmqpConnectionHolder.cs

```diff
@@ -31,22 +31,22 @@ public AmqpConnectionHolder(DeviceIdentity deviceIdentity)
         }
 
         public AmqpUnit CreateAmqpUnit(
-            DeviceIdentity deviceIdentity, 
-            Func<MethodRequestInternal, Task> methodHandler, 
-            Action<Twin, string, TwinCollection> twinMessageListener, 
+            DeviceIdentity deviceIdentity,
+            Func<MethodRequestInternal, Task> methodHandler,
+            Action<Twin, string, TwinCollection> twinMessageListener,
             Func<string, Message, Task> eventListener,
             Action onUnitDisconnected)
         {
             if (Logging.IsEnabled) Logging.Enter(this, deviceIdentity, $"{nameof(CreateAmqpUnit)}");
 
-            AmqpUnit amqpUnit = new AmqpUnit(
-                deviceIdentity, 
+            var amqpUnit = new AmqpUnit(
+                deviceIdentity,
```

Comment:
why var?

Reply:
we do a `new AmqpUnit()` here, so the type can be automatically inferred.